src/server/detail/wrapped_json_path.h

```diff
@@ -59,19 +59,34 @@ template <typename T> void OptionalEmplace(bool keep_defined, T src, T* dest) {
 
 }  // namespace details
 
+enum class JsonPathType { V2, Legacy /*Or V1*/ };
+constexpr JsonPathType kDefaultJsonPathType = JsonPathType::V2;
+
+struct CallbackResultOptions {
+  enum class SavingOrder { SAVE_FIRST, SAVE_LAST };
+  enum class OnEmpty { SEND_NIL, SEND_WRONGTYPE };
+
+  static constexpr SavingOrder kDefaultSavingOrder = SavingOrder::SAVE_LAST;
+  static constexpr OnEmpty kDefaultOnEmpty = OnEmpty::SEND_WRONGTYPE;
```

Comment:
I'd remove these and just assign default values directly in the initializers, IMHO it's more confusing than helpful

Reply:
Ok, but I'd like to keep `kDefaultJsonPathType`, because we use it in several places